demo/native_plots/bar_plot_demo.py

```diff
@@ -0,0 +1,87 @@
+import gradio as gr
+import pandas as pd
+
+from vega_datasets import data
+
+barley = data.barley()
+simple = pd.DataFrame({
+    'a': ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I'],
+    'b': [28, 55, 43, 91, 81, 53, 19, 87, 52]
+})
+
+def bar_plot_fn(display):
+    if display == "simple":
+        return gr.BarPlot.update(
+            simple,
+            x="a",
+            y="b",
+            title="Simple Bar Plot with made up data",
+            tooltip=['a', 'b'],
+            y_lim=[20, 100]
+        )
+    elif display == "stacked":
+        return gr.BarPlot.update(
+            barley,
+            x="variety",
+            y="yield",
+            color="site",
+            title="Barley Yield Data",
```

Comment:
Just wondering, should this one have a tooltip as well?

Reply:
Added! Good catch